### DISCUSSION TITLE
How to install the local .pack file?

### DISCUSSION THREAD

**Discussion by Qing201** [pyocd/pyOCD#1597] [Q&A]
I have a board that is not officially supported by pyOCD, but I have its .pack file, so I flash this board by using --pack arguments to use my local .pack file. I am looking for a way to install the local .pack file as built-in targets so I don't need to use --pack every time.

It seems there was a way to install the local .pack file using `pyocd pack -install-local`, but it is not working anymore. (answers from chatGPT), Is there a new way to install the local .pack file?

Thank you!!

**Comment by joaoricardo000** [comment]
up
